pkg/api/api.go

```diff
@@ -194,25 +195,26 @@ type Options struct {
 }
 
 type ExtraOptions struct {
-	Pingpong         pingpong.Interface
-	TopologyDriver   topology.Driver
-	LightNodes       *lightnode.Container
-	Accounting       accounting.Interface
-	Pseudosettle     settlement.Interface
-	Swap             swap.Interface
-	Chequebook       chequebook.Service
-	BlockTime        *big.Int
-	Tags             *tags.Tags
-	Storer           storage.Storer
-	Resolver         resolver.Interface
-	Pss              pss.Interface
-	TraversalService traversal.Traverser
-	Pinning          pinning.Interface
-	FeedFactory      feeds.Factory
-	Post             postage.Service
-	PostageContract  postagecontract.Interface
-	Steward          steward.Interface
-	SyncStatus       func() (bool, error)
+	Pingpong               pingpong.Interface
+	TopologyDriver         topology.Driver
+	LightNodes             *lightnode.Container
+	Accounting             accounting.Interface
+	Pseudosettle           settlement.Interface
+	Swap                   swap.Interface
+	Chequebook             chequebook.Service
+	BlockTime              *big.Int
+	Tags                   *tags.Tags
+	Storer                 storage.Storer
+	Resolver               resolver.Interface
+	Pss                    pss.Interface
+	TraversalService       traversal.Traverser
+	Pinning                pinning.Interface
+	FeedFactory            feeds.Factory
+	Post                   postage.Service
+	PostageContract        postagecontract.Interface
+	Steward                steward.Interface
+	SyncStatus             func() (bool, error)
+	RedistributionContract contract.Interface
```

Comment:
why is this needed ?

Reply:
because incentives would need this interface to make calls to contract.